flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobUtilsTest.java

```diff
@@ -22,26 +22,29 @@
 import static org.junit.Assume.assumeTrue;
 import static org.mockito.Mockito.mock;
 
-import com.google.common.io.Files;
-
 import org.apache.flink.util.OperatingSystem;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.Rule;
 
 import java.io.File;
 import java.io.IOException;
+import org.junit.rules.TemporaryFolder;
 
 public class BlobUtilsTest {
 
 	private final static String CANNOT_CREATE_THIS = "cannot-create-this";
 
 	private File blobUtilsTestDirectory;
 
+	@Rule
+	public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
 	@Before
-	public void before() {
+	public void before() throws IOException {
 		// Prepare test directory
-		blobUtilsTestDirectory = Files.createTempDir();
+		blobUtilsTestDirectory = temporaryFolder.newFolder();
```

Comment:
Minor issue: Would be nice to move this after the OS check, keep the directory operations logically together.